packages/client/modules/userDashboard/components/OrgBilling/OrgPlans.tsx

```diff
@@ -156,23 +159,58 @@ const UpgradeButton = styled(FlatPrimaryButton)<{
   }
 }))
 
-const OrgPlans = () => {
+const getButtonStyle = (tier: TierEnum, plan: TierEnum) => {
+  if (tier === 'starter') {
+    return plan === 'starter' ? 'disabled' : plan === 'team' ? 'primary' : 'secondary'
+  } else if (tier === 'team') {
+    return plan === 'team' ? 'disabled' : 'secondary'
+  } else {
+    return plan === 'enterprise' ? 'disabled' : 'secondary'
+  }
+}
+
+const getButtonLabel = (tier: TierEnum, plan: TierEnum) => {
+  if (tier === 'starter') {
+    return plan === 'starter' ? 'Current Plan' : plan === 'team' ? 'Select Plan' : 'Contact'
+  } else if (tier === 'team') {
+    return plan === 'team' ? 'Current Plan' : plan === 'starter' ? 'Downgrade' : 'Contact'
+  } else {
+    return plan === 'enterprise' ? 'Current Plan' : 'Contact'
```

Comment:
+1 Feels like could be simplified:

Something like this:

```
if (tier === plan) { 
  return 'Current Plan' 
} else if  (plan === 'enterprise') {
 return 'Contact'
} else if (plan === 'starter') {
return 'Downgrade'
} else { ... }
```

Reply:
We have to check both `plan` and `tier`, so I don't think that'll work unfortunately

Reply:
@nickoferrall `Current Plan` is always when `plan === tier`. So could be simplified like this:

```
  if (tier === plan) {
    return 'Current Plan'
  }

  if (tier === 'starter') {
    return plan === 'team' ? 'Select Plan' : 'Contact'
  } else if (tier === 'team') {
    return plan === 'starter' ? 'Downgrade' : 'Contact'
  } else {
    return 'Contact'
  }
```

But if not `current plan`, then enterprise is always `contact` as you can't downgrade to enterprise. If `starter` and not `current plan` then it is always donwgrade. 

So would be like this:

```
const getButtonLabel = (tier: TierEnum, plan: TierEnum) => {
  if (tier === plan) {
    return 'Current Plan'
  } else if (plan === 'enterprise') {
    return 'Contact'
  } else if (plan === 'starter') {
    return 'Downgrade'
  } else {
    return 'Select Plan'
  }
}
```

Does it make sense?